IsraelHiking.Web/src/application/components/dialogs/intro-dialog.component.html

```diff
@@ -8,22 +8,30 @@ <h2>{{resources.language}}</h2>
                 </div>
             </mat-radio-group>
             <br />
-            <img src="content/default-poi.png" class="img-responsive" />
+            <div class="squareContainer">
+                <ng-lottie class="img-responsive" style="padding-right: 13px;" [options]="lottieLanguage"></ng-lottie>
```

Comment:
`padding-right` will also work for English?

Reply:
yes